invenio_records_lom/records/mappings/os-v2/__init__.py

```diff
@@ -1,8 +1,8 @@
 # -*- coding: utf-8 -*-
 #
-# Copyright (C) 2021 Graz University of Technology.
+# Copyright (C) 2022 Graz University of Technology.
 #
 # invenio-records-lom is free software; you can redistribute it and/or modify it
 # under the terms of the MIT License; see LICENSE file for more details.
 
-"""Elasticsearch v7.* mappings for LOM records and drafts."""
+"""Opensearch .2* mappings for LOM records and drafts."""
```

Comment:
do you really mean version `.2*` here?
(sometimes this is short for `0.2*`, I think you mean version `2.*`)

Reply:
yes, 2.* is meant. i will change that.